src/pages/stop/stop.html

```diff
@@ -96,7 +96,7 @@
         </div>
       </div>
     </div>
-    <div *ngIf="order == '1'">
+    <div *ngIf="order == 'time'">
```

Comment:
The strings '0' and '1' (ah, strings of numbers) seemed arbitrary. I figured we should use the actual words, why not.